docs/technical_reference/unit_models/lt_med_surrogate.rst

```diff
@@ -0,0 +1,82 @@
+Low Temperature - Multi-effect Distillation (LT-MED)
+====================
+This Low Temperature Multi-effect Distillation (LT-MED) unit model
+   * supports steady-state only
+   * is a surrogate model
+   * is verified against the operation data in Plataforma Solar de Almeria (PSA)
+
+#TODO: Add index/reference to home page
```

Comment:
```suggestion
.. #TODO: Add index/reference to home page
```
This will show up in documentation. Instead, there should be something like `..` in front to indicate a comment with an .rst.

Reply:
Modified.